docs/references/strategies/languages/nodejs/npm-lockfile.md

```diff
@@ -44,3 +44,28 @@ The `requires` field signifies all of the dependencies that are needed by the de
 The `dependencies` field signifies all of the dependencies included in `babel-code-frame`'s `node_modules` folder within the top level `node_modules` folder. Notice that these dependencies are not always included in the `requires` section.
 
 > Note: `npm-shrinkwrap.json` is an identically formatted file that can be used for [publishing packages](https://docs.npmjs.com/cli/shrinkwrap).
+
+### Peer Dependencies
```

Comment:
I'm writing this up, but I'm not sure why not modeling peer deps directly is more user friendly. Is it because most package managers don't have them and it would be clutter?

Reply:
We discussed this offline, short version for posterity is that we could theoretically taken some other approach to handling peer deps, but this approach has the most pragmatic use for the user:

"Direct deps" are, in essence, "deps specified in `pacakge.json`".  Since peer deps are also specified there, and our only choices for modeling dep location are `direct` and `deep`, it makes sense to add them as direct.  If the dep somehow needs to be modified, that's the first place a user would have to look.  Any other peer deps are deep, because they simply aren't specified in the `package.json`.  That part is both technically true and pragmatically useful (or at least not harmful).

> but I'm not sure why not modeling peer deps directly is more user friendly
We just DON'T model peer deps in FOSSA at all.  Most tools don't have this concept.  I don't actually know if there's even a single other tool that does this.  I meant to say that the "other theoretical approach" was less user-friendly.

Reply:
I've added to the docs, I think I capture everything but please let me know if it needs more work.